_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "Index management UI enhancements"
+authors:
+- ihailong
+- gbinlong
+- suzhou
+- lxuesong
+date: 2023-02-28
+categories:
+ - technical-post
+meta_keywords: index management, index mappings, index operation UI, shrink index operations, OpenSearch Dashboards
+meta_description: Simplify cluster operations with index management UI enhancements that enable you to open, close, reindex, shrink, and split indexes in a more user-friendly way.
+
+excerpt: OpenSearch now offers users a new interface for running common indexing and data stream operations. Users can perform create, read, update, and delete (CRUD) and mapping for indexes, index templates, and aliases through the UI as well as open, close, reindex, shrink, and split indexes. This is the first step toward establishing an Index Management UI, which will serve as a unified administration panel in OpenSearch Dashboards.
+---
+
+Users may find it difficult to manage indexes, aliases, and templates on Opensearch Dashboard using only APIs. There are restrictions and relation-binding tasks, like adding aliases to indexes, simulating an index template by its name and more. We are excited to announce that these cluster operations have now been largely simplified with index management UI enhancements on Opensearch Dashboard v2.5.
+
+### Visual editor for index mappings with nested tree editor
+
+It is more complicated to build a JSON when it comes to index mappings, which have multiple nested layers and properties. To simplify that, a visual editor is provided with editable capabilities for nested properties. Users can add properties by clicking the operation buttons and then see what the mappings will look like by switching to the JSON editor.
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/index-mapping-editor.png" alt="index mapping editor"/>{: .img-fluid }
```

Comment:
Introduce the image in the text that precedes it. Often, ", as shown in the following image" will suffice.